### ISSUE TITLE
sdcv arguments?

### ISSUE THREAD

**Issue by rafaqz** [phongvcao/vim-stardict#5]
Hi thanks for the plugin. I'm trying load a specific dictionary using :StarDict -u "My dictionary" and :StarDict --data-dir /my/data/dir but neither really work, though simple flags like -h, -l and - u name_without_parenthesis do work.

It would make sense for arguments to :StarDict  to be passed verbatim to sdcv if vim can manage that.

**Comment by phongvcao** [comment]
The command `:StarDict --data-dir /my/data/dir "my word"` is working fine for me.

The reason `:StarDict -u "My Dictionary" "my word"` doesn't work maybe due to the sdcv prompt which lets you choose which dictionary to use (in case there are duplicate dictionaries). This is the command output of `sdcv -u "My Dictionary" "my word"` in my case:

``` bash
phongvcao@localhost$ sdcv -u "star_vietanh" anh
Found 2 items, similar to anh.
0)star_vietanh-->anh
1)star_vietanh-->anh
Your choice[-1 to abort]: 0
-->star_vietanh
-->anh

@anh
* noun
- Elder brother
=anh ruột+Blood elder brother
=anh cả+eldest brother
=anh cùng cha khác mẹ (hoặc cùng mẹ khác cha)+half-brother
=anh rể+brother-in-law (one's elder sister's husband)
=anh chồng+brother-in-law (one's husband's elder brother)
=anh vợ+brother-in-law (one's wife's elder brother)
- First cousin, cousin german (son of one's father's or mother's elder brother or sister)
=anh con nhà bác
```

Can you post here the output of your sdcv -u "My Dictionary" "my word" on command-line (Bash)? Thank you.

**Comment by rafaqz** [comment]
I ended up writing my own simple sdcv call and it works fine, so its not
the sdcv command...

Try to run:

:StarDict --data-dir ~/.stardict/folder -u "Bookname with spaces" word

**Comment by phongvcao** [comment]
@rafaqz can you give me like a download link to the dictionary you're using with sdcv so I can inspect this issue further? My dictionary's bookname actually doesn't contain any space in between. Thank you.

**Comment by phongvcao** [comment]
Fixed via ca3c04474efee67ab2bbc99c147a7a57715fb237

**Comment by phongvcao** [comment]
@rafaqz okay so from what I've found it is a Python piping issue. Your commands should now be working.
